Makefile

```diff
@@ -66,12 +89,14 @@ metadata: lib all
 
 test: all
 	$(MAKE) -C haskell test PACKAGE=segmented-cfmm \
-		SEGMENTED_CFMM_PATH=../$(OUT)/segmented_cfmm_default.tz
+		SEGMENTED_CFMM_PATH=../$(OUT)/segmented_cfmm_default.tz \
+		STORAGE_PATH=../$(OUT)/storage_default.tz
```

Comment:
... and remove the env var settings entirely.
```suggestion
```
(Note: same for the other haskell-related Makefile targets)

How does that sound? Does it seem like a good solution to you?

Reply:
Seems good to me, I've remove these env vars now.